### PR TITLE
Dispose participant view after stopping video streaming

### PR DESCRIPTION
**Pull request by cheukchuen**
## Purpose

* This PR fixes the green video indicator being displayed even after turning off the video. By disposing participant view ~before~ after stopping video streaming, the indicator now changes to orange if video is off.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
Verify that the following are valid
* During call, dot indicator should be green if video is on
* During call, dot indicator should be orange if video is off

### CODE REVIEW

**Comment by jimchou-dev** [comment]
Tested work on the callingView, can you also dispose it on the LobbyView?

File: AzureCalling/Controllers/CallViewController.swift [hunk 1]
```diff
@@ -262,6 +262,7 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
     @IBAction func onToggleVideo(_ sender: UIButton) {
         sender.isSelected.toggle()
         if sender.isSelected {
+            localParticipantView.dispose()
```
Comment (AmyL219) [inline AzureCalling/Controllers/CallViewController.swift:265]: Shouldn't we dispose it only after we get a success call back from StopLocalVideoStream?

Comment (cheukchuen) [reply AzureCalling/Controllers/CallViewController.swift:265]: Tried that, it's going to have the flickering behavior as stated [here](https://teams.microsoft.com/l/message/19:611ad8e0660848af8f3c80ee02f49ea6@thread.tacv2/1626824687747?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=8fac7e34-7319-4e89-95a1-c97cfe6a7344&parentMessageId=1626731965985&teamName=ACS%20Mobile%20UI&channelName=Calling%20Sample%20App&createdTime=1626824687747)

Comment (cheukchuen) [reply AzureCalling/Controllers/CallViewController.swift]: Gave some thought about it, made sense to only dispose after StopLocalVideoStream succeed, don't want any issue if we dispose the view and then encountered an error when StopLocalVideoStream failed.

Indicator flickering might not be a big issue, probably can ask the SDK team about it.

**Comment by cheukchuen** [comment]
> Tested work on the callingView, can you also dispose it on the LobbyView?

Added disposal in lobby

**Review by AmyL219 — APPROVED** [review]

**Review by jimchou-dev — APPROVED** [review]
Tested LGTM

**Pull request merged**